docs/src/main/asciidoc/security-guide.adoc

```diff
@@ -134,29 +135,109 @@ which permit all, deny all and only allow authenticated users respectively.
 
 If a path is registered with multiple permission sets then any permission sets that specify a HTTP method will take
 precedence and permissions sets without a method will not be considered (assuming of course the method matches). In this
-instance permission sets will only come into effect if the request method does not match any of the sets with method permissions.
-If a request would have matched based on path, but does not match any permission sets because of method requirements
-then this request is rejected.
+instance, the permission sets without methods will only come into effect if the request method does not match any of the
+sets with method permissions.
+
+[source,properties]
+--
+quarkus.http.auth.permission.permit1.paths=/public/*
+quarkus.http.auth.permission.permit1.policy=permit
+quarkus.http.auth.permission.permit1.methods=GET,HEAD
+
+quarkus.http.auth.permission.deny1.paths=/public/*
+quarkus.http.auth.permission.deny1.policy=deny
+--
+
+TIP: Given the above permission set, `GET /public/foo` would match both permission sets' paths,
+but because it matches the `permit1` permission set's explicit method, `permit1` will be chosen and the request will
+be accepted. `PUT /public/foo` on the other hand, will not match the method permissions of `permit1` and so
+`deny1` will be activated and reject the request.
+
+### Matching multiple paths and methods: both win
 
 If multiple permission sets specify the same path and method (or multiple have no method) then both permissions have to
 allow access for the request to proceed. Note that for this to happen both have to either have specified the method, or
-have no method, method specific matches take precedence as stated above.
+have no method, method specific matches take precedence as stated above:
+
+[source,properties]
+--
+
+quarkus.http.auth.policy.user-policy1.roles-allowed=user
+quarkus.http.auth.policy.admin-policy1.roles-allowed=admin
+
+quarkus.http.auth.permission.roles1.paths=/api/*,/restricted/*
+quarkus.http.auth.permission.roles1.policy=user-policy1
+
+quarkus.http.auth.permission.roles2.paths=/api/*,/admin/*
+quarkus.http.auth.permission.roles2.policy=admin-policy1
+--
+
+TIP: Given the above permission set, `GET /api/foo` would match both permission sets' paths,
+so would require both the `user` and `admin` roles.
 
+## Authenticated representation
+
+For every authenticated resource, you can inject an `SecurityIdentity` instance to get the authenticated identity information.
+
+In some other contexts you may have other parallel representations of the same information (or parts of it) such as `SecurityContext`
+for JAX-RS or `JsonWebToken` for JWT.
+
+## Configuration
+
+There are two configuration settings that alter the RBAC behavior:
+
+- `quarkus.jaxrs.deny-uncovered=true|false` - if set to true, the access will be denied for all JAX-RS endpoints will be
+denied by default. That is if the security annotations security annotations do not define the access control.
```

Comment:
```suggestion
denied by default. That is if the security annotations do not define the access control.
```

The previous sentence also needs to be corrected, there's too much denial in it :)

Reply:
I've fixed it.